### PR TITLE
unpacked img size

### PR DESCRIPTION
**Pull request by fahedouch**
fixing https://github.com/containerd/nerdctl/issues/50
Considering that the image unpacked size is the  image ChainID snapshot size

Signed-off-by: fahed dorgaa <fahed.dorgaa@gmail.com>

### CODE REVIEW

File: images.go [hunk 1]
```diff
@@ -94,7 +95,7 @@ func printImages(ctx context.Context, clicontext *cli.Context, imageList []image
 	}
 
 	for _, img := range imageList {
-		size, err := img.Size(ctx, cs, platforms.DefaultStrict())
+		size, err := unpackedImageSize(ctx, clicontext, client, img)
 		if err != nil {
 			return errors.Wrap(err, "failed to compute image size")
```
Comment (AkihiroSuda) [inline images.go:100]: This should be logrus.Warn, not return err

Comment (AkihiroSuda) [reply images.go:100]: (Multiple images may face this warning depending on snapshotter, so a warning should be printed only once)

Comment (fahedouch) [reply images.go]: @AkihiroSuda changes applied

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]
LGTM, but needs squashing

**Review by AkihiroSuda — APPROVED** [review]

**Pull request merged**